lib/nfs_common.pm

```diff
@@ -22,12 +22,19 @@ sub server_configure_network {
     configure_static_ip('10.0.2.101/24');
     configure_static_dns(get_host_resolv_conf());
 
-    if (is_sle('15+')) {
+    if (is_sle('15+') || is_opensuse) {
```

Comment:
shoulf this condition be removed?

Reply:
> shoulf this condition be removed?

No, as discussed we will enable it for TW in development group, then enable it when it the bug is fixed, the fix being on its way already.